base/docs/helpdb/Base.jl

```diff
@@ -564,9 +564,10 @@ julia> ones(Complex128, 2, 3)
 ones(t,dims)
 
 """
-    ones(A)
+    ones(A::AbstractArray, T=eltype(A)::Type, dims=size(A)::Tuple)
```

Comment:
There are at least some doctests where `dims...` is used. I can be more explicit about it.